src/gui/widgets/listbox.hpp

```diff
@@ -23,7 +23,6 @@
 
 #include "preferences/general.hpp"
 
-#include <boost/dynamic_bitset.hpp>
```

Comment:
My feeling is to leave this include here, and maybe optimise elsewhere.

This one is needed for this file, but is indirectly included via generator.hpp since 667064a8af6d. It's used there for a single function which looks like it could move to listbox.cpp instead. If that was done, generator.hpp wouldn't need to depend on Boost, and generator.hpp is included in a lot of files.